specifyweb/frontend/js_src/lib/components/Formatters/Definitions.tsx

```diff
@@ -68,7 +68,7 @@ export function Definitions({
         />
       ))}
       {!isReadOnly && hasCondition ? (
-        <div>
+        <div className="pt-4">
```

Comment:
Instead of padding, could you add gap-4 tot he parent element?

we generally use gap for spacing as it’s less bug prone than adding (gap is symmetrical regardless of how many children are inserted, where as padding is trickier to get right as you have to account for first or last element not needing padding)
We generally use padding for things like making buttons larger or adding symmetrical white space from  all directions in a container

Reply:
Due to `divide-y`, `gap-4` makes things really... unaesthetic.

<img width="400" alt="image" src="https://github.com/specify/specify7/assets/37256050/944bbeb3-e22f-4bad-8d7a-e031162e19ae">
<img width="400" alt="image" src="https://github.com/specify/specify7/assets/37256050/b7723c2f-c324-455d-a217-5790c79a145d">

I wanted to do this initially but settled on the current solution. Thoughts?

Reply:
Due to `divide-y`, `gap-4` makes things really... unaesthetic.

<img width="400" alt="image" src="https://github.com/specify/specify7/assets/37256050/944bbeb3-e22f-4bad-8d7a-e031162e19ae">
<img width="400" alt="image" src="https://github.com/specify/specify7/assets/37256050/b7723c2f-c324-455d-a217-5790c79a145d">

I wanted to do this initially but settled on the current solution. Thoughts?

Reply:
I'll mark this as resolved for now, but if you have a solution in mind in the future we can open a new PR and resolve it. Thank you @maxpatiiuk !

Reply:
ah good point, I forgot about the divide
this is the best solution then, and similar to what we do in the workbench mapper and query builder
the other option is to insert a div with height of 1 in between each item to simulate "border", but your solution is simpler